setup.py

```diff
@@ -28,9 +28,8 @@
     long_description = f.read()
 
 package_name = "dbt-glue"
-package_version = "0.2.15"
-dbt_min_version = "1.2.0"
-dbt_max_version = "1.3.2"
+package_version = "1.3.0"
```

Comment:
personal opinion, this is quite a big bump that is not motivating the increase the major version.
How about using 0.3.0? I guess that quite some other features need to be released before we reach a total final stable version 1.0.0. Also following the semantic version, I will suggest increasing only the minor version, and keeping still the major to 0.

Reply:
agreed